docs/sources/setup-grafana/configure-docker.md

```diff
@@ -8,45 +8,70 @@ keywords:
   - configuration
   - documentation
   - docker
+  - docker compose
 title: Configure Docker image
 weight: 1800
 ---
 
 # Configure a Grafana Docker image
 
-If you are running Grafana in a Docker image, then you configure Grafana using [environment variables]({{< relref "./configure-grafana/#override-configuration-with-environment-variables" >}}) rather than directly editing the configuration file. If you want to save your data, then you also need to designate persistent storage or bind mounts for the Grafana container.
+Running Grafana on Docker may require some additional configuration which depends on the complexity of your environment. We will explain some of those e.g. using different images, changing logging levels, defining secrets on Cloud (e.g. AWS), configuring plugins etc.
```

Comment:
```suggestion
Running Grafana on Docker might require some additional configuration depending on your environment's complexity. We will explain some of those e.g. using different images, changing logging levels, defining secrets on Cloud (e.g. AWS), configuring plugins etc.
```

Reply:
what do you mean by "explain some of those"? Explain some of what?

Also, we want to avoid use of the word 'we'. You can simply say "This topic includes/explains etc."

Reply:
I cannot explain the "some of those" as already defined as an e.g. in the text i.e. e.g. using different images, changing logging levels, defining secrets on Cloud (e.g. AWS), configuring plugins etc.

Yes, agree to remove "we" from the above.

We can talk more on this one via chat

Reply:
Ok, I updated, please take a look.